src/Files.App/ServicesImplementation/NetworkDrivesService.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+using Files.App.DataModels.NavigationControlItems;
+using Files.App.Shell;
+using Files.Backend.Services;
+using Files.Sdk.Storage.LocatableStorage;
+using Vanara.PInvoke;
+using Vanara.Windows.Shell;
+
+namespace Files.App.ServicesImplementation
+{
+	public class NetworkDrivesService : INetworkDrivesService
+	{
+		public bool DisconnectNetworkDrive(ILocatableFolder drive)
```

Comment:
Small nit, but not that important: I'd be nice if you could add "<inheritdoc />" to signify that a method is documented 🙂